services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -38,6 +38,9 @@ class WriteMetadataActor(override val batchSize: Int,
       case Success(_) =>
         putWithResponse foreach { case (ev, replyTo) => replyTo ! MetadataWriteSuccess(ev) }
       case Failure(regerts) =>
+        val workflowMetadataFailureCounts = e.toVector.flatMap(_.events).groupBy(x => x.key.workflowId).map { case (wfid, list) => s"$wfid: ${list.size}" }
+        log.error("Metadata events permanently dropped for the following workflows: " + workflowMetadataFailureCounts.mkString(","))
```

Comment:
I feel like "dropped" doesn't quite do justice to the situation because sometimes things get dropped deliberately
```suggestion
        log.error("Metadata events irretrievably lost for the following workflows: " + workflowMetadataFailureCounts.mkString(","))
```

Reply:
> sometimes things get dropped deliberately

Under what circumstances?

Reply:
One can deliberately drop a DB table during a migration - this was, perhaps confusingly, a comment on the word "drop" rather than on dropping metadata

Reply:
got it, I was indeed confused

Reply:
@cjllanwarne you have my blessing to completely ignore this comment so as to avoid running tests again